pkgs/by-name/ra/radicle-desktop/package.nix

```
@@ -0,0 +1,127 @@
+{
+  fetchFromGitHub,
+  cargo-tauri,
+  fetchgit,
+  git,
+  glib,
+  gtk3,
+  importNpmLock,
+  lib,
+  libsoup_3,
+  nodejs,
+  openssh,
+  openssl,
+  pkg-config,
+  playwright-driver,
+  radicle-node,
+  rustPlatform,
+  webkitgtk_4_1,
+  wrapGAppsHook4,
+}:
+
+let
+  twemoji-assets = fetchFromGitHub {
+    owner = "twitter";
+    repo = "twemoji";
+    tag = "v14.0.2";
+    hash = "sha256-YoOnZ5uVukzi/6bLi22Y8U5TpplPzB7ji42l+/ys5xI=";
+  };
+in
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "radicle-desktop";
+  version = "0.8.0";
+
+  src = fetchgit {
+    url = "https://seed.radicle.xyz/z4D5UCArafTzTQpDZNQRuqswh3ury.git";
+    rev = "aeb405aaf53b56a426ab8d68c7f89b8953683224";
+    hash = "sha256-DdjmE4Jgf1oGqJpVF7piBtCjt07pb7er1ZaFaVYr+wY=";
+    leaveDotGit = true;
+    postFetch = ''
+      git -C $out rev-parse HEAD > $out/.git_head
+      git -C $out log -1 --pretty=%ct HEAD > $out/.git_time
+      rm -rf $out/.git
+    '';
+  };
+
+  npmDeps = importNpmLock {
+    version = finalAttrs.version;
+    pname = finalAttrs.pname + "-npm-deps";
+    npmRoot = finalAttrs.src;
+  };
```

I think this may solve the evaluation error

```suggestion
  npmDeps = importNpmLock {
    version = finalAttrs.version;
    pname = finalAttrs.pname + "-npm-deps";
    npmRoot = finalAttrs.src;
  };
  
  npmConfigHook = importNpmLock.npmConfigHook;
```